src/main/java/net/sf/jabref/event/ChangeSpecialFieldEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+public class ChangeSpecialFieldEvent {
```

Comment:
This class can be removed. See https://github.com/JabRef/jabref/pull/1028/files#r62419955